lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -19,7 +20,7 @@
 @attr('shard_3')
 @ddt.ddt
 @patch.dict('django.conf.settings.FEATURES', {'ENABLE_SPECIAL_EXAMS': True, 'MILESTONES_APP': True})
-class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin):
+class MilestonesTransformerTestCase(CourseStructureTestCase, MilestonesTestCaseMixin, ModuleStoreTestCase):
```

Comment:
Hrm... I wouldn't think that would be an issue, since all of the the subclasses would appear earlier in the MRO. As long as all of the subclasses (which we control) call `super`, then it should still work.